examples/webExample/package.json

```diff
@@ -3,17 +3,17 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "dev": "webpack-dev-server --content-base dist/ --open",
+    "dev": "webpack-dev-server --open",
     "watch": "webpack -w",
     "build": "webpack"
   },
   "keywords": [],
   "author": "developers@mparticle.com",
   "license": "ISC",
   "devDependencies": {
-    "webpack": "^4.32.2",
-    "webpack-cli": "^3.3.2",
-    "webpack-dev-server": "^3.11.0"
+    "webpack": "^5.90.0",
```

Comment:
the web example had some very old dependencies which, while benign since it's just an example project, were lighting up our dependabot. this change could technically be separate from this branch.